src/main/java/Main.java

```diff
@@ -1,5 +1,5 @@
 public class Main {
     public static void main(String[] args) {
-//ыщьу сщьь
+//developing 2
```

Comment:
```suggestion
//developing 3
```
sorry, it should be developing 3 also